test_conformance/api/test_kernels.cpp

```diff
@@ -704,3 +713,43 @@ REGISTER_TEST(negative_set_immutable_memory_to_writeable_kernel_arg)
 
     return TEST_PASS;
 }
+
+REGISTER_TEST(negative_invalid_arg_size)
```

Comment:
Probably want "sampler" in the name here somewhere, e.g.

```suggestion
REGISTER_TEST(negative_invalid_arg_size_sampler)
```